src/core/components/radio/Radio.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { ReactNode, InputHTMLAttributes } from 'react';
+import {
+	label,
+	labelWithSupportingText,
+	radio,
+	labelText,
+	labelTextWithSupportingText,
+	supportingText,
+} from './styles';
+import { Props } from '@guardian/src-helpers';
+
+const LabelText = ({
+	hasSupportingText,
+	children,
+}: {
+	hasSupportingText?: boolean;
+	children: ReactNode;
+}) => {
+	return (
+		<div
+			css={(theme) => [
+				hasSupportingText ? labelTextWithSupportingText : '',
+				labelText(theme.radio && theme),
+			]}
+			className="src-radio-label-text"
+		>
+			{children}
+		</div>
+	);
+};
+
+const SupportingText = ({ children }: { children: ReactNode }) => {
+	return (
+		<div css={(theme) => supportingText(theme.radio && theme)}>
+			{children}
+		</div>
+	);
+};
+
+export interface RadioProps
+	extends InputHTMLAttributes<HTMLInputElement>,
+		Props {
+	/**
+	 * TODO: Add a description or remove as value is inherited from `InputHTMLAttributes<HTMLInputElement>` (as `string | number | readonly string[] | undefined`)
+	 */
+	value: string;
```

Comment:
Any thoughts on whether we should add a description or remove this prop and accept the inherited type instead?

Reply:
I'd say remove it. It's not adding any specific value – unless we want to document _all_ props that are inherited from  `InputHTMLAttributes<HTMLInputElement>` 😅

Reply:
Done in https://github.com/guardian/source/pull/902/commits/7dd7f80c4260edd6a3228a1250940a61742a0b65